build/repo.targets

```diff
@@ -154,22 +122,26 @@
       <Solution>
         <AdditionalProperties>RestoreGraphOutputPath=$(_RestoreGraphSpecsDirectory)%(Solution.Repository)\%(Solution.FileName)%(Solution.Extension).json</AdditionalProperties>
       </Solution>
+
+      <GraphSpecInputs Include="
```

Comment:
Well, as long as we always force clean on CI we're okay. Still a little concerned as this will be hard to debug...but let's give it a try.